mechanisms/Gfluct3.mod

```diff
@@ -194,18 +194,6 @@ INITIAL {
        }
 }
 
-VERBATIM
-#include "nrnran123.h"
```

Comment:
@olupton : if we want these changes to be compatible with current master then we can't remove includes like this:

```console
 -> Compiling bgka.c
Gfluct3.c: In function 'mynormrand_Gfluct3':
Gfluct3.c:368:29: warning: implicit declaration of function 'nrnran123_normal' [-Wimplicit-function-declaration]
  368 |                         x = nrnran123_normal((nrnran123_State*)_p_donotuse);
      |                             ^~~~~~~~~~~~~~~~
Gfluct3.c:368:47: error: 'nrnran123_State' undeclared (first use in this function)
  368 |                         x = nrnran123_normal((nrnran123_State*)_p_donotuse);
```

Reply:
Right, this makes sense, but it's a bit annoying.

**Maybe** it would work to leave the `#include` and remove the declarations underneath it, or maybe this would be a good use of https://github.com/neuronsimulator/nrn/issues/1696, or maybe it's not worth the effort to be compatible with both versions at once...

Reply:
I hesitate to suggest this, but `__cplusplus` would also let us add conditional code for these cases as an interim measure.